deployment/createconstants.ps1

```diff
@@ -0,0 +1,46 @@
+# Powershell script for VSTS build that accesses build environment variables to create config file.
```

Comment:
Does this replace the other createconstants.ps1? Do we have a reason to maintain two versions if it doesn't?

Reply:
Yes. Whence we have EventUI Build working, cfg/createconstants.ps1 will be removed.